Snakefile

```diff
@@ -58,10 +58,10 @@ rule clean:
 # Rules for intermediate steps
 rule table_costs_efficiencies:
     input: "raw/scalars/costs_efficiencies.csv"
-    output: "results/_tables/technical_and_cost_assumptions_{scenario_key}.csv"
+    output: "results/_tables/technical_and_cost_assumptions_{scenario}.csv"
```

Comment:
`scenario_key` was intended because raw data is labeled by `scenario_key`s. Scenarios are formed by combining several scenario_keys. Can you revert these changes?

Reply:
Ok, good to know. I have reverted that commit.